src/roman_datamodels/stnode.py

```diff
@@ -553,6 +553,17 @@ def from_yaml_tree(self, node, tag, ctx):
         return force_roman_unit(u.Unit(node, parse_strict="silent"))
 
 
+class Associations(dict):
+    __slots__ = ()
+    _tag = ""
+
+    def __getattr__(self, attr):
+        return self[attr]
+
+    def __setattr__(self, attr, value):
+        self[attr] = value
```

Comment:
I suggest we remove this class if not needed now

Reply:
Will do.